docs/index.rst

```diff
@@ -28,9 +28,15 @@ Astropy Documentation
 
     .. image:: _static/astropy_logo.pdf
 
-Welcome to the Astropy documentation! Astropy is a community-driven
-package intended to contain much of the core functionality and some common
-tools needed for performing astronomy and astrophysics with Python.
+The ``astropy`` core package is a community-driven package which contains key
```

Comment:
@taldcroft - this is right under the logo which includes "A community python library for astronomy" as its subtitle. I think we should try to avoid repeating that, and also keep this part very short, so it doesn't distract from the actually useful links (most people will need to read this only once, but will look at the links many times). So¸ how about more simply:
```
``astropy`` contains key functionality and common tools needed for
astronomy and astrophysics. It is at the core of the ... 
[rest is great]
```